### DISCUSSION TITLE
[博客数量上线] 最大博客数量999这个限制能够接触吗？

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#2058] [Ideas]
**为什么提出这个新的特性改动**
简要说明此特性解决的问题，例如，『博客站点的读者互动性不够强，和读者无法建立紧密的联系...』

**描述一下你推荐的解决方案**
简要说明你的解决方案建议，例如，『Giscus评论插件功能更加强大，用户只需留言既可在你的邮箱收到通知。。。』

**描述一下你考虑过的其它替代解决方案**
简要说明你所有想过的有可能解决此问题的方案。

**补充说明**
补充与此特性相关的内容

**Comment by tangly1024** [comment]
这个是由于NotionAPI的接口最多请求1000个数据；超出的会被忽略，后续会考虑解决方案。

可能需要多数据库版本的Notion才能解决，但是此功能的使用频率极低
